foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-StoragePolicySetDefinition.json

```diff
@@ -263,7 +263,7 @@
                             }
                         }
                     },
-                    "name": "Deny-Storage-LocalUser"
+                    "name": "Deny-Storage-Sftp"
```

Comment:
Is changing the name of the definition and updating the parameter references in the policySet really necessary?

Reply:
Yes. This name is more accurate.

Reply:
Please revert back as this will break the existing policySet when making an update for those who have already deployed, and requires the assignment to 1) be removed, 2) remove the policySet, 3) redeploy the new policySet, and 4) make a new policyAssignment using the new policySet